index.html

```diff
@@ -641,15 +641,15 @@ <h2>Core Data Model</h2>
       <p class="note"
         title="Abstract concepts vs. concrete concepts">
 Readers might note that some abstract concepts described in this section
-do not have normative requirements or associated media types defined by this
-specification. For example, the abstract concepts of a <a>credential</a> and a
+do not have normative requirements nor associated media types defined by this
+specification. For example, the abstract concepts of <a>credential</a> and
 <a>presentation</a> do not have normative requirements nor associated media
 types. However, the concrete concepts of a <a>verifiable credential</a> or a
 <a>verifiable presentation</a> are defined as <a>conforming documents</a> and
-do have associated media types. Furthermore, what differentiates the
-abstract concepts of <a>credential</a> and <a>presentation</a> from the
-concrete concepts of a <a>verifiable credential</a> and a
-<a>verifiable presentation</a> is the fact that only the "verifiable" objects
+do have associated media types. Furthermore, the only difference between the
```

Comment:
I read this again, and the word only is misleading here

The list of differences: `credentials` are abstract, have no serialization, and no media type, `verifiable credentials` are concrete, have serializations, and have media types. `verifiable credentials` have "abstracts proofs / cryptographic assurances"... `credentials` to do not.

I feel we are already saying this, but I also feel it could be made clearer, without leaning so heavily on "abstracts proofs / cryptographic assurances" as the dividing line.

Reply:
Yes, I agree with the language above. I'll try another reword in a future pass that tries to simplify the language. I specifically like the language you provided here: "credentials are abstract, have no serialization, and no media type, verifiable credentials are concrete, have serializations, and have media types. verifiable credentials have "abstracts proofs / cryptographic assurances"... credentialsdo not." ... I will try to use a variation of that without losing other things that previous other reviewers wanted in the text as well.

Reply:
> You have to produce a serialization of a credential to feed it to a mechanism that will output a verifiable credential including internal or external proof.

Yes to this. We need to be careful with the language such that we don't make this impossible or "questionably wrong" in some way. This is why I alluded to "verifiable credential" just being "the name" we happen to have chosen to refer to a document that conforms to the VCDM.

It's important that we say that there are mechanisms that can be used to secure a VC such that it becomes "cryptographically verifiable", but we should not create some language that makes it impossible / incorrect to, for example, send a VC (with `application/vc+ld+json` as the media type) to a service that will secure it in some way.

Reply:
> This is why I alluded to "verifiable credential" just being "the name" we happen to have chosen to refer to a document that conforms to the VCDM.

The problem with this, is we know there are several competing concrete syntaxes for the name "verifiable credential"...  so we are choosing a name, and an approach that is making this worse, not better, when we create further confusion regarding `credential` vs `verifiable credential`.

> but we should not create some language that makes it impossible / incorrect to, for example, send a VC (with application/vc+ld+json as the media type) to a service that will secure it in some way.

This works fine, if you assume `verifiable credential` is a document format, not a securing format.

If you assert that a `verifiable credential` has at least one layer of cryptography applied to it... but you also assert that the media type works for serializations that have 0 layers of cryptography around it.... we have just reinvented "alg: none" for JSON.

The problem appears to be media type confusion born of the fact that data integrity proofs do not have a separate media type.

We don't have this problem for JWT / SD-JWT etc... because they have media types and that are secured representations, that secure JSON payloads (ignoring the fact that "alg: none" still exists there).

They treat the content / payload as unsecured (explicitly in the case of "alg:none")... they wrap it with an external proof, and you end with 2 media types:

1. `application/vc+ld+json` a JSON media type with possibly no security
1. `application/jwt` / `application/sd-jwt`  / `application/vc+ld+json+sd-jwt` a string encoding of a media type that secured a JSON claimset, (possibly with no security, aka "alg: none").

With Data integrity Proofs, we have the following:

1. `application/vc+ld+json` a JSON media type with possibly no security (proof is optional)
1. `application/vc+ld+json` a JSON media type with security (proof is mandatory)

"alg: none" happens when you use the same representation for both secured and unsecured content... data integrity proofs are creating this issue, by riding along inside the core data model's media type, based on the presence of a JSON member.... that is exactly what happens when you decode the "protected header" and find "alg:none" instead of "alg:ES384".... its presence of a member, determining if the media type is actually secured.

IMO, the best solution to this is to actually drop the "security language" from the core data model entirely.. just treat the core data model as a JSON-LD Document model, and don't have ANY langauge that implies anything about security (remove the `proof` concept from the core data model completely).

Then is the securing specifications, explain the concept of proof, is specific to the securing specification, and comment on its unique properties...

For example: 
- JWT has "alg:none", is this allowed for VC?
- Data Integrity Proofs have "proof sets and proof chains", are these allowed for VCs?

etc... the problem remains that we are doing too much security work in the core data model, and that leads to bias towards specific security formats, like data integrity proofs... or weak language, caused by trying to leave the door open for each security specification to handle things differently...  (embedded vs external proofs).